app/assets/stylesheets/modules/image-collection-filmstrip.scss

```diff
@@ -82,6 +54,13 @@
 
       li:last-child {
         margin-right: 0;
+        width:100px;
+        height:100px;
+        padding:27px;
+        text-align: center;
+        span {
+            color:#666;
+        }
```

Comment:
In the JS the size of the parent is set dynamically by the image size. There may be some opportunity to refactor how that is done to a more css-based approach in the future. Also, see: https://github.com/sul-dlss/SearchWorks/issues/1185